app/views/members/activities/partials/_view.html.haml

```diff
@@ -45,6 +45,12 @@
           \-
           = activity.end_time.strftime("%H:%M")
       <br/>
+      - if activity.open_date != nil && activity.open_date > Date.today
```

Comment:
A quick google suggests to me that `Date.today` in rails shouldn't really be used, [please validate](http://winstonyw.com/2014/03/03/time_now_vs_time_zone_now/). I haven't checked which time/function is used for the check to see if someone can register for an activity but these need to be the same and I would suggest creating a helper function in ruby

Reply:
We use `Date.today` everywhere for activities, so I think it'd be best to stick with that?

Reply:
consistency 🫠